benchmark/ordering.py

```diff
@@ -52,7 +47,7 @@ def main():
             max_intersection = 0
             matching_idx = 0
             for pred_box in order_pred.bboxes:
-                intersection = pred_box.intersection_pct(Bbox(bbox=bbox))
+                intersection = pred_box.intersection_pct(PolygonBox(polygon=bbox))
```

Comment:
I think this might throw an error, we have a PolygonBox.from_bbox() classmethod in Marker for cases like this
https://github.com/VikParuchuri/marker/blob/3a29cd71a366551d626f33301f67e5c3ed3d90da/marker/schema/polygon.py#L186-L193

Reply:
Polygonbox can take a bbox or a polygon now